<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgIntegration</class>
 <widget class="QDialog" name="dlgIntegration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>211</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lao UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Infinity Calculator &gt;&gt; Integration</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/resources/resources/windowicon.png</selectedon>
   </iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="txtFormula">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: #e8e8e8;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFormula">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Formula</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtMinRange">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: #e8e8e8;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMinRange">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>65</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Min Range</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtMaxRange">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>60</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: #e8e8e8;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMaxRange">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>65</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max Range</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClose">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 	background-color: #dbdbdb;
    border: 2px;
	color:#464747;

}

QPushButton:hover{background-color:#40bd9e; color:white;}</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIntegrationVar">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Integration&lt;br/&gt;Variable&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbIntegrationVar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOk">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 	background-color: #dbdbdb;
    border: 2px;
	color:#464747;

}

QPushButton:hover{background-color:#40bd9e; color:white;}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
